face/AndroidDetect/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.0.0'
+        classpath 'com.android.tools.build:gradle:8.3.2'
```

Comment:
Using AGP 8.0.0 results in the following error:

```console
Execution failed for task ':app:mergeExtDexDebug'.
java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null
```

With the build environment:
* Android Studio | Ladybug (2024.2.1) September 2024
* JDK 21, which is the default JetBrains Runtime (JBR) that comes with Android Studio Ladybug
* Gradle 8.5, the minimum version which supports Java 21 (see [compatibility matrix](https://docs.gradle.org/current/userguide/compatibility.html))

---

Upgrading AGP from 8.0.0 to 8.3.2, which is again the minimum version compatible with Gradle 8.5 (see [compatibility 
 table](https://developer.android.com/build/releases/gradle-plugin#updating-gradle) and [8.3.* release notes](https://developer.android.com/build/releases/past-releases/agp-8-3-0-release-notes)), solves the problem.

Accordingly, the API level should bump from 33 to 34.